Automattic/delphin
# Remove Privacy Protection toggle

Fixes #807, #911, & #931. Requires D3694-code.

After some brief discussion (and it was discussed somewhere else previously too), we decided to remove the Privacy Protection toggle and line item from the checkout screen. Reasons as follows:

* As an included feature we offer free of charge, it doesn't need to be in the checkout summary
* We are not required to allow users to disable it. If we're pitching it as a feature, let's go all in.
* We disable it by default for TMCH registrations
* There were some cases we discussed during prototyping why you might want it to be public, but they felt weak

Since the privacy toggle is removed, we now mention keeping their info private on the Contact Information page where it's more contextual.

Before | After
------------ | -------------
<img width="464" alt="screen shot 2016-12-09 at 16 37 48" src="https://cloud.githubusercontent.com/assets/448298/21065721/d53988ba-be2f-11e6-822d-eac7184f365e.png"> | <img width="467" alt="screen shot 2016-12-09 at 16 31 43" src="https://cloud.githubusercontent.com/assets/448298/21065725/dc572cc4-be2f-11e6-8ea0-705b9f645cd2.png">

The new copy on contact information form:

<img width="470" alt="screen shot 2016-12-09 at 16 22 16" src="https://cloud.githubusercontent.com/assets/448298/21065735/e700acc2-be2f-11e6-875a-b7ff9ab13a7b.png">

#### Testing

* Apply D3694-code to your sandbox.
* Go through checkout flow with a non-TM domain
* Assert the new copy appears at the bottom of the contact information form
* Assert the privacy toggle switch and total cost are both removed from the order summary
* Now go through checkout flow with a TM domain
* Assert the new copy on `/contact-information` mentions the required public whois
* Make sure the checkout process still works as expected

#### Review


cc @ranh for copy review on contact info screen
